backend/src/api/files.py

```
@@ -42,17 +50,49 @@ class FileStatsResponse(BaseModel):
 async def upload_file(
     file: UploadFile = File(...),
     title: str = Form(...),
+    description: Optional[str] = Form(None),
     tags: Optional[str] = Form(None),
     is_public: bool = Form(False),
-    current_user: User = Depends(can_upload_documents),
+    processing_priority: Optional[str] = Form('normal'),
+    enable_quality_check: bool = Form(True),
+    custom_metadata: Optional[str] = Form(None),
```

Ruff (0.14.1)

Unused function argument: `custom_metadata`

(ARG001)